test/token/ERC20BridgedPermit.unit.test.ts

```diff
@@ -149,6 +187,12 @@ unit("ERC20BridgedPermit", ctxFactory)
     );
 
     assert.equalBN(await erc20BridgedProxied.getContractVersion(), 2);
+
+    // can't initialize after finalizeUpgrade_v2
```

Comment:
:warning:  do we have a test that we can't use `initialize` instead of `finalizeUpgrade_v2` for the mainnet upgrade scenario?